snapshot/snapshot.go

```diff
@@ -0,0 +1,33 @@
+// Modifications Copyright 2021 The klaytn Authors
+// Copyright 2020 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+//
+// This file is derived from core/state/snapshot/snapshot.go (2021/10/21).
+// Modified and improved for the klaytn development.
+package snapshot
+
+import (
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/crypto"
+)
+
+var (
+	// emptyRoot is the known root hash of an empty trie.
+	emptyRoot = common.HexToHash("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")
```

Comment:
I think this values are defined somewhere else. Do we need define this again here?

Reply:
The `emptyRoot` are defined multiple times and those are private. I think those should be updated as well. How about refactoring this later?

Reply:
got it.